app/services/policies_config_params_wrapper.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class PoliciesConfigParamsWrapper
```

Comment:
Maybe Wrapper is not the right word, my fault, I induced you to write this with my comment. Trying to find a replacement...

How about simply `PoliciesConfigParams` and use `call` instead of `wrap`?

Reply:
sure, done.